index.template.html

```diff
@@ -1168,7 +1168,10 @@ <h2>Class Definitions</h2>
         </p>
 
         <p>
-            Please note that all vocabulary terms and values are case sensitive.
+            <span class="rfc2119-assertion" id="td-processor">
+                Please note that all vocabulary terms and values MUST be case sensitive.
```

Comment:
Does this really hold for values?

Are 10m different from 10M ?

Reply:
@mlagally I think the point you are raising goes beyond what the TD can offer. You are asking for a *real* canonicalization process.
I guess no-one would argue that besides case-sensitive charterers we would also need to match the number "1" with "1.0" etc. Hence, also values need to match "exactly"...

Reply:
I think the case sensitive requirement is different to the canonicalization one.  Canonicalization will be a topic for 2.0.